helpers/data-exchange-payload-generator.js

```diff
@@ -0,0 +1,135 @@
+
+
+export class DataExchangePayloadGenerator {
+    itemsAmount
+
+    constructor(uuid, itemsAmount = 1000) {
+        this.itemsAmount = itemsAmount
+        this.uuid = uuid
+    }
+
+    prepareProductsForUpdate(recentlyCreatedProducts) {
+        function updateFunc(product) {
+            let updateConfig = {
+                productAbstractProducts: {
+                    is_quantity_splittable: false,
+                    productSearch: {
+                        is_searchable: false
+                    },
+                    productStocks: {
+                        quantity: Math.round(Math.random() * 1000)
+                    },
+                    productLocalizedAttributes: {
+                        description:  `productLocalizedAttributes Updated at: ${new Date().toISOString()}`
+                    },
+                    
+                },
+                productAbstractPriceProducts: {
+                    price: Math.round(Math.random() * 100),
+                    priceProductStores:{
+                        id_price_product_store: 'toInt',
+                        priceProductStoreDefaults: {
+                            id_price_product_default: "toInt",
+                            fk_price_product_store: "toInt"
+                        }
+                    }
+                },
+                productAbstractLocalizedAttributes: {
+                    description:  `productAbstractLocalizedAttributes Updated at: ${new Date().toISOString()}`
+                },
+            }
+
+            for (const firstLevelKey of Object.keys(updateConfig)) {
+                let secondLevel = updateConfig[firstLevelKey]
+                for (const secondLevelKey of Object.keys(secondLevel)) {
+                    if (typeof secondLevel[secondLevelKey] === 'object') {
+                        for (const thirdLevelKey of Object.keys(secondLevel[secondLevelKey])) {
+                            product[firstLevelKey].map((el) => {
+                                return el[secondLevelKey].map((key) => {
+                                    if (Array.isArray(key[thirdLevelKey])) {
+                                        key[thirdLevelKey] = key[thirdLevelKey].map((tr) => {
+                                            for (const forthLevelKey of Object.keys(tr)) {
+                                                if (forthLevelKey in secondLevel[secondLevelKey][thirdLevelKey]) {
+                                                    if (secondLevel[secondLevelKey][thirdLevelKey][forthLevelKey] === 'toInt') {
+                                                        tr[forthLevelKey] = parseInt(tr[forthLevelKey])
+                                                    } else {
+                                                        tr[forthLevelKey] = secondLevel[secondLevelKey][thirdLevelKey][forthLevelKey]
+                                                    }
```

Comment:
It is very difficult to read but I don't require any changes since we don't have any conventions for this repo, up to you